pages/osx/cot.md

```diff
@@ -0,0 +1,24 @@
+# cot
+
+> The Plain-Text Editor for macOS.
+> More information: <https://coteditor.com/>.
+
+- Start CotEditor:
+
+`cot`
+
+- Open a new blank document:
+
+`cot --new`
+
+- Open specific files:
+
+`cot {{path/to/file1 path/to/file2 ...}}`
+
+- Open a specific file in the foreground (blocking the terminal until the editor is closed):
+
+`cot --wait {{path/to/file}}`
```

Comment:
```suggestion
- Open specific files:

`cot --wait {{path/to/file1 path/to/file2 ...}}`
```

Reply:
Should this still keep some information about how this command blocks the terminal until the editor is closed? Maybe "Open specific files (blocking the terminal until the editor is closed)"?

Reply:
I don't think so. Anyway we have to decide whether to write such info everywhere where appropriate or not. 😄

Reply:
I think we should keep the previous version. We somehow have to explain what the `--wait` option does. Otherwise this example would be useless.